src/playwright/transformPlaywrightJson.test.ts

```diff
@@ -37,24 +37,33 @@ describe('Playwright Json', () => {
                   title: \\"Example/Header\\",
                   name: \\"Logged In\\"
                 };
-                page.on('pageerror', err => {
+                const pageErrorListener = err => {
                   page.evaluate(({
                     id,
                     err
                   }) => __throwError(id, err), {
                     id: \\"example-header--logged-in\\",
                     err: err.message
                   });
-                });
+                };
+                page.on('pageerror', pageErrorListener);
```

Comment:
you can make this change and remove `page.off('pageerror', pageErrorListener);`, it should give the same effect!
```suggestion
                page.once('pageerror', pageErrorListener);
```